crates/admin/src/cluster_controller/service.rs

```diff
@@ -698,6 +698,7 @@ mod tests {
 
     struct NodeStateHandler {
         persisted_lsn: Arc<AtomicU64>,
+        archived_lsn: Arc<AtomicU64>,
```

Comment:
We don't currently update these from anywhere that I can see (apart from tests) but this handler returns `PartitionProcessorStatus`, so it seemed fitting to add the `archived_lsn` attribute.